libpam0g-dev is required

**Version info**

Ubuntu 22.04
shpool v0.6.2


**Logs**

```


error: failed to run custom build command for `motd v0.2.1`                                                          
                                                                                                                     

Caused by:                                                                                                           
  process didn't exit successfully: `/tmp/cargo-installfLaNPd/release/build/motd-0e127c12db4c87b5/build-script-build`(exit status: 101)                                                                                                   
  --- stdout                                                                                                         
  cargo:rerun-if-changed=src/pam_motd_overlay.c                                                                      
  cargo:rerun-if-changed=src/pam_motd_overlay_versions.ldscript                                                      
                                                                                                                     
  --- stderr                                                                                                         
  thread 'main' panicked at /home/damien/.cargo/registry/src/index.crates.io-6f17d22bba15001f/motd-0.2.1/build.rs:23:3:                                                                                                                   
  build failed: error building overlay, code = exit status: 1                                                        
  STDOUT:                                                                                                            
  STDERR: ./src/pam_motd_overlay.c:11:10: fatal error: security/_pam_types.h: No such file or directory              
     11 | #include <security/_pam_types.h>                                                                           
        |          ^~~~~~~~~~~~~~~~~~~~~~~                                                                           
  compilation terminated.                                                                                            
                                                                                                                     
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                      
warning: build failed, waiting for other jobs to finish...                                                           
error: failed to compile `shpool v0.6.2`, intermediate artifacts can be found at `/tmp/cargo-installfLaNPd`.         
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Solution**

sudo apt-get install libpam0g-dev

--- a/libshpool/Cargo.toml
+++ b/libshpool/Cargo.toml
@@ -36,7 +36,7 @@ tracing = "0.1" # logging and performance monitoring facade
 bincode = "1" # serialization for the control protocol
 shpool_vt100 = "0.1.2" # terminal emulation for the scrollback buffer
 shell-words = "1" # parsing the -c/--cmd argument
-motd = { version = "0.2.1", default-features = false, features = [] } # getting the message-of-the-day
+motd = { version = "0.2.2", default-features = false, features = [] } # getting the message-of-the-day
 termini = "1.0.0" # terminfo database
 tempfile = "3" # RAII tmp files
 strip-ansi-escapes = "0.2.0" # cleaning up strings for pager display

